src/controls/MyTeams/IMyTeamsProps.ts

```diff
@@ -0,0 +1,26 @@
+import { IReadonlyTheme } from "@microsoft/sp-component-base";
+import { WebPartContext } from "@microsoft/sp-webpart-base";
+export interface IMyTeamsProps {
+  /**
+    /**
+     * Title of MyTeams
+     */
+  title?: string;
+  /**
+   *  themeVariant
+   */
+  themeVariant?: IReadonlyTheme;
+  /**
+   *  webPart COntext
+   */
+  webPartContext: WebPartContext;
```

Comment:
Could you please use `BaseComponentContext` instead of `WebPartContext`? 
In that case the component could be used in extensions as well.

Reply:
changed to BaseComponentContext